public/css/elements/stripBox.sass

```diff
@@ -30,10 +30,10 @@
 
   .stripBox__sectionContent
     margin: 0px 10px
-    box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.1)
-    border-left: 1px solid rgba(0, 0, 0, 0.1)
-    border-right: 1px solid rgba(0, 0, 0, 0.1)
-    border-bottom: 1px solid rgba(0, 0, 0, 0.1)
+    // box-shadow: 0px 0px 4px rgba(0, 0, 0, 0.1)
```

Comment:
I think Baćko wants to have it there for reference, at least until it is all finished